create_infotexts.py

```diff
@@ -381,9 +374,114 @@ def process_depicted_place(self, place_string, places_mapping):
             self.meta_cats.append("Media_contributed_by_SMVK_without_mapped_place_value")
             place_as_wikitext = place_string
 
-
         self.data["depicted_place"] = place_as_wikitext
 
+    def generate_list_of_stripped_keywords(self, keyword_string):
+        """
+        Transform string of keywords from column <Nyckelord> to list of keywords.
+
+        :param keyword_string: String from column <Nyckelord.
+        :return: list of keywords.
+        """
+        keywords_list = keyword_string.split(", ")
+        if "Svenska Cypernexpeditionen" in keywords_list:
+            keywords_list.remove("Svenska Cypernexpeditionen")
+
+        self.data["Nyckelord"] = keywords_list
```

Comment:
I would call this something like "keyword_list" in the `self.data` dictionary as well